Representable/representable
# Profanity check v2

added filter to screen for profanity-- must install the google client library for python which can be done by typing the following into terminal:

`pip install --upgrade google-api-python-client google-auth-httplib2 google-auth-oauthlib`
note that we will also need to install this on the representable server

Testing TODO:

- Install library `pip install --upgrade google-api-python-client google-auth-httplib2 google-auth-oauthlib`
- Add to environment file (type `nano ~/.bash_profile` into Terminal) the following line
`export PERSPECTIVE_API_KEY = "<key that i will send to you securely>"` save and restart Terminal
- make map using profane language in any of the descriptions, name, or title
- check if a report was made either in Postico or Django admin
- make map using non-profane language in all descriptions
- make sure a report hasn't been issued, either in Django Admin or Postico
